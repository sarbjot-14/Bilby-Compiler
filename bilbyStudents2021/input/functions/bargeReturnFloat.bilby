func float barge (int e, float f, int g ) {
	print e $n;
	
	return f;
}

main{
	print "before call" $n;
	imm ans  := barge(5,6.7,8);
	print ans $n;
}