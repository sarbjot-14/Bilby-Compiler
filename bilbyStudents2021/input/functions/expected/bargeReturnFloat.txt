before call
5
6.700000
